### ISSUE TITLE
keras tuner lack of documentation for the  Objective parameter values

### ISSUE THREAD

**Issue by 21kc-caracol** [keras-team/keras-tuner#588]
In the following example:
`            tuner = kt.Hyperband(model,
                                 objective='val_accuracy',
                                 max_epochs=10,
                                 factor=3,
                                 directory='my_dir',
                                 project_name='intro_to_kt')`

I can't find the documentation for all the possible `objective` values, can anyone help?

I've searched in the official documentation and in stackOverflow, etc.

**Comment by haifeng-jin** [comment]
Please refer to the keras metrics page: https://keras.io/api/metrics/
If it is a metric function, the name is the function name.
If it is a `Metric` subclass, the name is in the `__init__` function.

**Comment by 21kc-caracol** [comment]
Can you show the example of where val_accuracy is defined in the above link?

I have entered into your link and tried to find it for sanity purposes...

Thanks in advance!

**Comment by haifeng-jin** [comment]
The link is a complete list of metrics classes. It does not provide the string values.
You can pass any of the class instances to your model. And set it as the objective.

`objective("val_metric_name", direction="min")`

Replace the "metric_name" with the name arg value in the metric class initializer.

We have a tutorial for this on the way.

**Comment by 21kc-caracol** [comment]
sounds good, how can i follow the tutorials release?

Things begin to be clearer, but i still would have want an example for knowing from where exactly i took the 'val_accuracy' string. I mean- from which class/subclass on the page you shared.

Thanks a lot!

**Comment by haifeng-jin** [comment]
The string is always the snake case of the class name.
For example, `'accuracy'` is from `Accuracy`.

It seems there is no way to follow the update of the website.
It will be released regularly. The tutorial is in the master branch of the website now.
